dist/images/start-ovs-dpdk-v2.sh

```diff
@@ -103,11 +103,12 @@ esac
 
 ovs-vsctl set port ${DPDK_TUNNEL_IFACE} tag=${VLAN_TAG}
 
-ip addr add ${ENCAP_IP} dev ${DPDK_TUNNEL_IFACE}
 fi
 
 ip link set ${DPDK_TUNNEL_IFACE} up
 
+ip addr add ${ENCAP_IP} dev ${DPDK_TUNNEL_IFACE}
```

Comment:
`add` will report error if the address already exists when just restart the pod, we should use `replace` instead

Reply:
> 
ack， i will change it